test/jdk/javax/swing/DefaultListCellRenderer/4180943/bug4180943.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2000, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 4180943
+ * @summary Extra borders created by DefaultListCellRenderer
+ * @run main bug4180943
+ */
+
+import javax.swing.DefaultListCellRenderer;
+
+public class bug4180943 {
+    public static void main(String[] argv) {
+        DefaultListCellRenderer lcr1 = new DefaultListCellRenderer();
```

Comment:
In this case not critical to the test, but I think it would be a good example for others to perform Swing related code on EDT.
(applicable to all tests).